cxplat/cxplat_test/cxplat_memory_test.cpp

```diff
@@ -16,14 +16,18 @@ TEST_CASE("allocate", "[memory]")
     // Try an allocation that need not be cache aligned.
     uint64_t* buffer = (uint64_t*)cxplat_allocate(CxPlatNonPagedPoolNx, 8, TEST_TAG, false);
     REQUIRE(buffer != nullptr);
+#ifndef NDEBUG
     REQUIRE(*buffer != 0);
+#endif
```

Comment:
Worth defining a wrapper for `REQUIRE` instead?

Reply:
Didn't seem worth it yet, but if I add more, then I will.